config/database.yml

```diff
@@ -2,7 +2,7 @@ default: &default
   adapter:  postgresql
   host:     localhost
   encoding: unicode
-  pool:     10
+  pool:     20
```

Comment:
Why would you increase the pool? Usually, unless you use an external connection pooler, increasing the pool's size leads to poorer performance. As far as I see, Good Job does not recommend it either https://github.com/bensheldon/good_job#database-connections